content/08.transmission.md

```diff
@@ -142,21 +142,31 @@ The sequencing data can be found at GISAID [@https://www.gisaid.org/], NCBI [@ht
 
 A number of studies have suggested associations between individuals' racial and ethnic backgrounds and their COVID-19 risk.
 In particular, Black Americans are consistently identified as carrying a higher burden of COVID-19 than white Americans [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1], with differences in the rates of kidney complications from COVID-19 particularly pronounced [@doi:10.1016/j.kint.2020.05.006].
+Statistics from a number of cities indicate significant discrepancies between the proportion of COVID-19 cases and especially deaths in Black Americans relative to their representation in the general population [@doi:10/ggv494].
```

Comment:
What is especially highlighting here? I think it could just be _cases and deaths_...